samba-manager/files/samba-manager.init

```diff
@@ -96,8 +96,7 @@ parse_config() {
 	config_get password ${cfg} password
 	config_get sharename ${cfg} sharename
 
-	mount=$(get_mount)
-	if [ $enabled -eq 1 -a ! -z $mount ] ; then
+	if [ $enabled -eq 1 ] ; then
 		add_samba_user $username $password
 		generate_samba_config $username $sharename $mount
```

Comment:
Looking at this again, how does mount get set to anything in this function?  I'm guessing this code doesn't actually work the way it is.  We may still need to call get_mount in this function to actually get the mount name to pass to generate_samba_config